### PR TITLE
Display latency info for API calls 

### PR DESCRIPTION
**Pull request by shrutichidam**
Display the elapsed time from calling the BE API to receiving a response. 
Modify the `Read` method to call the BE API as well so it can track the elapsed time.

image

### CODE REVIEW

File: src/Web/ViewModels/FraudProtectionIOModel.cs [hunk 1]
```diff
@@ -37,7 +37,8 @@ public void Add(object request, SampleResponseRaw response, string name = "", bo
             {
                 Request = skipSerialization ? request as string : JsonSerializer.Serialize(request, JsonSerializerOptions),
                 Response = JsonSerializer.Serialize(response.RawData, JsonSerializerOptions),
-                Name = name
+                Name = name,
+                CallTime = response.ResponseTime
```
Comment (dickwyn) [inline src/Web/ViewModels/FraudProtectionIOModel.cs:41]: nit: could we call this variable `ResponseTime`?

File: src/Web/Views/Shared/_FraudProtectionIO.cshtml [hunk 2]
```diff
@@ -35,6 +35,12 @@
                         </div>
                     </section>
                 </div>
+                @if (@io.CallTime > 0)
+                {
+                    <div class="row">
+                        <h5>The above call took @io.CallTime ms</h5>
```
Comment (dickwyn) [inline src/Web/Views/Shared/_FraudProtectionIO.cshtml:41]:
```suggestion
                        <h5>The response time for the above request is @io.CallTime ms</h5>
```

**Comment by chadsowald** [comment]
Can the code show the latency below/closer to the correlation ID so that all metadata about a call is together?

**Review by chadsowald — DISMISSED** [review]
Just left one comment about positioning/wording.  Looks great though!

**Comment by shrutichidam** [comment]
> Can the code show the latency below/closer to the correlation ID so that all metadata about a call is together? And, in that case, you might not need "above", "below". Just "API call duration" or something.

I had the latency where it is since I associated it with each request/response pair, but I can change it to be one overall

File: src/Web/ViewModels/FraudProtectionIOModel.cs (hunk 1 above)
Comment (shrutichidam) [reply src/Web/ViewModels/FraudProtectionIOModel.cs]: updated!

File: src/Web/Views/Shared/_FraudProtectionIO.cshtml (hunk 2 above)
Comment (shrutichidam) [reply src/Web/Views/Shared/_FraudProtectionIO.cshtml]: changed this to be API call duration based on Chad's comment below

**Comment by shrutichidam** [comment]
Updated version looks like: 
image
or this (when there's multiple requests):
image

**Review by chadsowald — APPROVED** [review]

**Pull request merged**